### DISCUSSION TITLE
Landing page not working ?

### DISCUSSION THREAD

**Discussion by davidszerman** [requarks/wiki#5626] [Help / Questions] [answered]
Hello

I have a question based on [this doc](https://docs.requarks.io/guide/structure).

> When creating multi-level pages, you may want to have a landing page for each of the virtual folders.
> For example, if you have a page at path /foo/bar, you should create a new page at path /foo. This will ensure that clicking the "foo" breadcrumbs link on the /foo/bar page leads to the /foo page.
> If you're using the Site Tree navigation mode (or Custom Navigation), the page title for each level is taken from its corresponding page.

I have created the page /Wiki/Armee/Armee_air/Grades

I then have created /Wiki/Armee/Armee and /Wiki/Armee (I tried both).

I thought that with this organization, when clicking the /Wiki/Armee folder, it would display the Armee page in the main window but it only displays the page list on the sidebar like in screenshot.

Did I miss something ?

(in the screnenshot "Armée" is only the title but the technical name of the file is "Armee")

Thank you

![armee](https://user-images.githubusercontent.com/27421517/189308012-aa906413-3350-41c9-9d85-3db0b39369d1.png)

![admin](https://user-images.githubusercontent.com/27421517/189366292-590391f9-a153-4922-bc1e-14c1bc861d45.png)

**Comment by NGPixel** [comment, marked as answer]
This is the current behavior. When clicking a folder, you then need to click the first link (above "Répertoire actuel") to access the folder page.

<img width="165" alt="image" src="https://user-images.githubusercontent.com/15522395/190262050-02a51c87-de5a-4de1-a06f-ececeecc5e16.png">

This will be fixed to work like you expected in the upcoming 3.0 version.
